bin_tests/tests/crashtracker_bin_test.rs

```diff
@@ -104,7 +104,8 @@ fn test_crash_tracking_bin(
     assert_eq!(
         serde_json::json!({
           "signum": 11,
-          "signame": "SIGSEGV"
+          "signame": "SIGSEGV",
+          "crash_address": "0x0000000000000000",
```

Comment:
faulting address ? Trying to find a name that disambiguates from the instruction pointer
I'll add a comment on RFC

Reply:
I don't mind renaming it, I think `faulting_address` works nicely as well.

Reply:
Renamed `crash_address` to `faulting_address` in https://github.com/DataDog/libdatadog/pull/663/commits/0d4bb98e732dce251a1d06a1ba5fa35b116fff4a